MCprep_addon/MCprep_resources/Languages/en/LC_MESSAGES/mcprep.po

```diff
@@ -0,0 +1,523 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
```

Comment:
We should probably fill these out/replace with the actual GPL header, or remove and have it just use the package's parent license inherently.